src/renderers/shared/ReactDebugTool.js

```diff
@@ -244,11 +301,13 @@ var ReactDebugTool = {
   onBeginLifeCycleTimer(debugID, timerType) {
     checkDebugID(debugID);
     emitEvent('onBeginLifeCycleTimer', debugID, timerType);
+    markBegin(debugID, timerType);
```

Comment:
So good, I was going to ask if you could make this less verbose and repetitive :)